crates/common_utils/src/errors.rs

```diff
@@ -29,11 +29,14 @@ pub enum ParsingError {
     /// Failed to parse email
     #[error("Failed to parse email")]
     EmailParsingError,
+    /// Failed to parse email
```

Comment:
```suggestion
    /// Failed to parse phone number
```

Reply:
@vlad-onis This minor change is pending.